static-site/src/sections/sars-cov-2-forecasts-page.jsx

```diff
@@ -0,0 +1,203 @@
+      }
+    }
+  },
+  {
+    title: "Estimated Cases over time",
+    text: (
+      <span>
+        As estimated by the variant renewal model.
+        These estimates are smoothed to deal with daily reporting noise and weekend effects present in case data.
+      </span>
+    ),
+    images: {
+      nextstrainClades: {
+        alt: "Global estimated variant case plots from GISAID data",
+        ...generatePictureSrcs(nextstrainCladesSrcBase, "smoothedIncidencePanel")
+      }
+    }
+  },
+  {
+    title: "Estimated effective reproduction number over time",
+    text: (
+      <span>
+        This is an estimate of the average number of secondary infections expected to be caused by an individual infected with a given variant as estimated by the variant renewal model.
+        In general, we expect the variant to be growing if this number is greater than 1.
+      </span>
+    ),
+    images: {
+      nextstrainClades: {
+        alt: "Global variant Rt plots from GISAID data",
+        ...generatePictureSrcs(nextstrainCladesSrcBase, "rtPanel")
+      }
+    }
+
+  }
+];
+
+
+function Index(props) {
+  const [ cladeType, setCladeType ] = useState("nextstrainClades");
+
+  return (
+    <GenericPage location={props.location} banner={DisclaimerBanner()}>
+      <splashStyles.H1>{title}</splashStyles.H1>
+      <SmallSpacer />
+
+      <FlexCenter>
+        <splashStyles.CenteredFocusParagraph>
+          {abstract}
+        </splashStyles.CenteredFocusParagraph>
+      </FlexCenter>
+      <MediumSpacer />
+
+      <PathogenPageIntroduction data={introContents} />
+      <HugeSpacer />
+
+      {collapsibleContents.map((c) => <CollapsibleContent key={c.title} content={c} cladeType={cladeType} />)}
+    </GenericPage>
+  );
+}
+
+const DisclaimerBanner = () => {
+  return (
+    <splashStyles.FixedBanner backgroundColor="#ffedcc">
+      <splashStyles.StrongerText>
+        {disclaimer}
+      </splashStyles.StrongerText>
+    </splashStyles.FixedBanner>
+  )
+}
+
+const FullWidthPicture = styled.picture`
+  width: 100%;
+  height: auto;
+  display: block;
+  text-align: center;
+`;
+
+function CollapsibleContent(props) {
+  /* eslint no-shadow: "off" */
+  const {title, text, images} = props.content;
+
+  return (
+    <Collapsible
+      triggerWhenOpen={<CollapseTitle name={title} isExpanded />}
+      trigger={<CollapseTitle name={title} />}
+      triggerStyle={{cursor: "pointer", textDecoration: "none"}}
+      open={true}
+    >
+      <div style={{ padding: "10px" }} >
+        <splashStyles.FocusParagraph>
+          {text}
+        </splashStyles.FocusParagraph>
+        <MediumSpacer />
+        <FullWidthPicture>
+          {images[props.cladeType]?.srcSets
+            ? images[props.cladeType].srcSets
+                .map((source) => <source key={source.srcset} media={source.media} srcSet={source.srcSet}/>)
+            : null
+          }
+          <img
```

Comment:
On iPhone, the screen width is 375px (ish) but the `_small.png` is 504px and it overflows. Setting `max-width: 100%;` on the `<img>` element scales this down (has to be on the `<img>` from my testing).

Otherwise the responsive images look great!

Reply:
Thanks for the tip! I updated the style in the force-push.